terraform/container-app/contentful.tf

```diff
@@ -0,0 +1,22 @@
+//Double check this is the right thing
+data "azurerm_cdn_frontdoor_endpoint" "app" {
+  name                = "s190d01-plantech-container-app-url"
+  profile_name        = "s190d01-plantech-cdnwaf"
+  resource_group_name = "s190d01-plantech"
+}
+
+resource "null_resource" "health_check" {
```

Comment:
why call it health check?

Reply:
Because I copy and pasted it from TF's docs before changing it, and I forgot to amend the name :) Good spot, fixed